packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -7698,7 +7718,9 @@ namespace Tpetra {
     {
       typename local_graph_type::row_block_type::HostMirror lclGraph_rbo_host_this = Kokkos::create_mirror_view(this->lclGraph_.row_block_offsets);
       typename local_graph_type::row_block_type::HostMirror lclGraph_rbo_host_graph = Kokkos::create_mirror_view(graph.lclGraph_.row_block_offsets);
+      // DEEP_COPY REVIEW - NOT TESTED
       Kokkos::deep_copy(lclGraph_rbo_host_this, this->lclGraph_.row_block_offsets);
+      // DEEP_COPY REVIEW - NOT TESTED
```

Comment:
I'm not sure what condition would exercise this branch.
This routine is used only for comparisons in swap test.